test_runner/fixtures/pageserver/allowed_errors.py

```diff
@@ -89,6 +89,8 @@ def scan_pageserver_log_for_errors(
     # During teardown, we stop the storage controller before the pageservers, so pageservers
     # can experience connection errors doing background deletion queue work.
     ".*WARN deletion backend: calling control plane generation validation API failed.*Connection refused.*",
+    # Can happen when the test shuts down the storage controller while it is calling the utilization API
+    ".*WARN.*path=/v1/utilization .*request was dropped before completing",a
```

Comment:
a is extra probably?

Reply:
This might be a very good candidate to move to global allowed_errors, but perhaps we could also shutdown in different order...